packages/bootstrap/README.md

```diff
@@ -0,0 +1,25 @@
+# @kaizen/bootstrap
+
+Bootstrap is a place for Kaizen-specific styles that cannot be contained within a component package (such as Button).
+
+Note that this package is _not_ for browser normalisations or polyfills. Instead refer to @cultureamp/frontend-runtime:
+https://github.com/cultureamp/frontend-ops/blob/master/packages/frontend-runtime/lib/index.ts
+
+This package exports two assets; a Sass file and a React component. For most use cases it is easiest to wrap your App in the React component: 
```

Comment:
Now that Elm is deprecated I'm going to drop the a Sass file recommendation, and steer people to use the React wrapper.